quakesaver_client/models/sensor.py

```diff
@@ -60,6 +64,82 @@ def get_measurement(self: Sensor, query: MeasurementQuery) -> MeasurementResult:
             raise CorruptedDataError() from e
         return result
 
+    def get_peak_ground_acceleration(
```

Comment:
Can we add that like that? That was a feature request from our colleagues in Montenegro.

```suggestion
def get_peak_horizontal_acceleration(
        self: Sensor, query: MeasurementQuery
    ) -> MeasurementResult:
        """Get the PHA measurement of the sensor.
        Args:
            query: The query parameters like time frame and aggregator.
        Returns:
            MeasurementQuery: The queried data (if exists) as time series.
        """
        full_query = MeasurementQueryFull(
            **query.dict(), field="pha", measurement="rt_peak_ground_motion"
        )
        return self._get_measurement(query=full_query)

    def get_peak_ground_acceleration(
```